addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.xml

```diff
@@ -17,17 +17,18 @@
                                 <button class="button print btn btn-lg btn-secondary w-100 py-3" t-on-click="() => doPrint.call()">
                                     <i t-attf-class="fa {{doPrint.status === 'loading' ? 'fa-fw fa-spin fa-circle-o-notch' : 'fa-print'}} me-1" />Print Receipt
                                 </button>
-                                <div class="d-flex gap-1" t-att-class="{'flex-column': this.ui.isSmall}">
-                                    <div class="flex-grow-1 p-0 border-0 position-relative send-receipt-input">
-                                        <input type="text" class="border p-3 bg-view w-100 pe-5" t-attf-placeholder="{{ state.mode === 'email' ? 'e.g. john.doe@mail.com' : 'e.g. 0123456789' }}" t-model="state.input" />
-                                        <div t-att-class="{'opacity-25': !isValidInput}"  t-on-click="() => this.actionSendReceipt()" class="position-absolute end-0 top-0 h-100 d-flex align-items-center text-primary cursor-pointer">
-                                            <i class="fa fa-arrow-circle-right fs-1 pe-3" aria-hidden="true" />
+                                <div class="d-flex flex-column gap-2">
+                                    <div class="d-flex">
+                                        <input type="text" class="flex-grow-1 border p-3 bg-view pe-5 border-end-0 rounded-start-2 position-relative send-receipt-email-input" t-attf-placeholder="e.g. john.doe@mail.com" t-model="state.email" />
+                                        <div>
+                                            <button t-att-style="`width: ${this.ui.isSmall ? '4rem' : '8rem'}`" class="btn btn-primary btn-lg lh-lg d-flex align-items-center justify-content-center rounded-start-0 h-100 send-receipt-on-email-btn" t-att-disabled="!isValidEmail" t-on-click="() => this.actionSendReceiptOnEmail()">
```

Comment:
Maybe here you can simplify a bit this button. I don't think you need this `d-flex align-items-center justify-content-center`. You don't need `send-receipt-on-email-btn`, in the test you can just look for `button i.fa-paper-plane`.

You should also be able to combine the icon and the button in one tag: `<button class="fa fa-paper-plane" />`